tests/validator/validator.cpp

```diff
@@ -836,62 +836,6 @@ TEST(Validator, validateInvalidConnectionsVariableWithoutParentComponent)
     EXPECT_EQ_ERRORS(expectedErrors, v);
```

Comment:
Should all of these `expectedError` strings be renamed to `e` as in earlier PR?

Reply:
The tests don't need to be identical in their coding style for me.

Reply:
And that's why I hate it: it's a mess.

Reply:
It needs to be different so you can try doing things differently, if everything is done the same you are not testing over a very wide base.

Reply:
We are talking about variable names here...

Reply:
I thought we were talking about all things to do with test coding style.  I don't have a problem with changing variable names.

Reply:
Re coding style in general, it's also a mess for me and I don't see how how it helps with testing over a wider base...? Anyway, those are "only" tests, I guess...